src/translator/scorers.cpp

```diff
@@ -1,24 +1,22 @@
 #include "translator/scorers.h"
+#include "common/io.h"
 
 namespace marian {
 
 Ptr<Scorer> scorerByType(const std::string& fname,
                          float weight,
                          const std::string& model,
-                         Ptr<Config> config) {
-  Ptr<Options> options = New<Options>();
-  options->merge(config);
+                         Ptr<Options> options) {
   options->set("inference", true);
```

Comment:
clone for cleanliness?

Reply:
IMHO clone here would be redundant as `scorerByType` is only used in `createScorers` using a newly cloned options specific for each scorer. 

To be able to test this I have added regression tests for ensembles of models with different architectures.